var/spack/repos/builtin/packages/ncdu/package.py

```diff
@@ -46,7 +46,7 @@ class Ncdu(Package):
 
     def install(self, spec, prefix):
         configure('--prefix=%s' % prefix,
-                  '--with-ncurses=%s' % spec['ncurses'])
+                  '--with-ncursesw=%s' % spec['ncurses'])
```

Comment:
ncursesw is the preferred version, but I don't see why it shouldn't be able to use both